### ISSUE TITLE
Exception when sending email to many recipients when pipelining is enabled

### ISSUE THREAD

**Issue by mhjelmseth** [jstedfast/MailKit#1498]
**Describe the bug**
We get an exception when sending emails to many recipients using our smtp server with pipelining enabled. When we disable pipelinging client side, everything works as expected. Is doesn't matter if we use a real smtp server or a local smtp server meant for development like Papercut SMTP - the returned error message is the same.

**Platform (please complete the following information):**
 - OS: Windows
 - .NET Runtime: CoreCLR
 - .NET Framework: .NET 6
 - MailKit Version: 3.4.3

**Exception**
System.ArgumentException
  HResult=0x80070057
  Message=The output byte buffer is too small to contain the encoded data, encoding 'Unicode (UTF-8)' fallback 'System.Text.EncoderReplacementFallback'. (Parameter 'bytes')
  Source=System.Private.CoreLib
  StackTrace:
   at System.Text.Encoding.ThrowBytesOverflow()
   at System.Text.Encoding.ThrowBytesOverflow(EncoderNLS encoder, Boolean nothingEncoded)
   at System.Text.Encoding.GetBytesWithFallback(ReadOnlySpan`1 chars, Int32 originalCharsLength, Span`1 bytes, Int32 originalBytesLength, EncoderNLS encoder)
   at System.Text.Encoding.GetBytesWithFallback(Char* pOriginalChars, Int32 originalCharCount, Byte* pOriginalBytes, Int32 originalByteCount, Int32 charsConsumedSoFar, Int32 bytesWrittenSoFar, EncoderNLS encoder)
   at System.Text.Encoding.GetBytes(Char* pChars, Int32 charCount, Byte* pBytes, Int32 byteCount, EncoderNLS encoder)
   at System.Text.EncoderNLS.Convert(Char* chars, Int32 charCount, Byte* bytes, Int32 byteCount, Boolean flush, Int32& charsUsed, Int32& bytesUsed, Boolean& completed)
   at MailKit.Net.Smtp.SmtpStream.TryQueueCommand(Encoder encoder, String command, Int32& index)
   at MailKit.Net.Smtp.SmtpStream.QueueCommand(String command, CancellationToken cancellationToken)
   at MailKit.Net.Smtp.SmtpClient.Send(FormatOptions options, MimeMessage message, MailboxAddress sender, IList`1 recipients, CancellationToken cancellationToken, ITransferProgress progress)
   at MailKit.Net.Smtp.SmtpClient.Send(FormatOptions options, MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
   at MailKit.MailTransport.Send(MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
   at Program.<Main>$(String[] args) in C:\Users\[username]\source\repos\MailKitTest\MailKitTest\Program.cs:line 38

**To Reproduce**
1. Paste code below into i.e. a console application and run.

**Expected behavior**
We expect the email to be sent without getting any errors.

**Code Snippets**
```csharp
var message = new MimeMessage();

message.From.Add(new MailboxAddress(string.Empty, "sender@tempuri.org"));

for (int i = 1; i <= 1000; i++)
{
    message.Bcc.Add(new MailboxAddress(string.Empty, $"receiver+{i}@tempuri.org"));
}

message.Subject = "Test message";

var bodyBuilder = new BodyBuilder
{
    HtmlBody = "Message content"
};

message.Body = bodyBuilder.ToMessageBody();

using var smtpClient = new SmtpClient(new ProtocolLogger("smtp.log"));
smtpClient.Connect("localhost");
smtpClient.Send(message);
```

**Protocol Logs**
Connected to smtp://localhost:25/?starttls=when-available
S: 220 Papercut.Service v9.0.0.0 ESMTP ready
C: EHLO DESKTOP-DK13FHT
S: 250-Papercut.Service Hello DESKTOP-DK13FHT, haven't we met before?
S: 250-PIPELINING
S: 250-8BITMIME
S: 250 SMTPUTF8
C: MAIL FROM:<sender@tempuri.org>
C: RCPT TO:<receiver+1@tempuri.org>
C: RCPT TO:<receiver+2@tempuri.org>
C: RCPT TO:<receiver+3@tempuri.org>
C: RCPT TO:<receiver+4@tempuri.org>
C: RCPT TO:<receiver+5@tempuri.org>
C: RCPT TO:<receiver+6@tempuri.org>
C: RCPT TO:<receiver+7@tempuri.org>
C: RCPT TO:<receiver+8@tempuri.org>
C: RCPT TO:<receiver+9@tempuri.org>
C: RCPT TO:<receiver+10@tempuri.org>
C: RCPT TO:<receiver+11@tempuri.org>
C: RCPT TO:<receiver+12@tempuri.org>
C: RCPT TO:<receiver+13@tempuri.org>
C: RCPT TO:<receiver+14@tempuri.org>
C: RCPT TO:<receiver+15@tempuri.org>
C: RCPT TO:<receiver+16@tempuri.org>
C: RCPT TO:<receiver+17@tempuri.org>
C: RCPT TO:<receiver+18@tempuri.org>
C: RCPT TO:<receiver+19@tempuri.org>
C: RCPT TO:<receiver+20@tempuri.org>
C: RCPT TO:<receiver+21@tempuri.org>
C: RCPT TO:<receiver+22@tempuri.org>
C: RCPT TO:<receiver+23@tempuri.org>
C: RCPT TO:<receiver+24@tempuri.org>
C: RCPT TO:<receiver+25@tempuri.org>
C: RCPT TO:<receiver+26@tempuri.org>
C: RCPT TO:<receiver+27@tempuri.org>
C: RCPT TO:<receiver+28@tempuri.org>
C: RCPT TO:<receiver+29@tempuri.org>
C: RCPT TO:<receiver+30@tempuri.org>
C: RCPT TO:<receiver+31@tempuri.org>
C: RCPT TO:<receiver+32@tempuri.org>
C: RCPT TO:<receiver+33@tempuri.org>
C: RCPT TO:<receiver+34@tempuri.org>
C: RCPT TO:<receiver+35@tempuri.org>
C: RCPT TO:<receiver+36@tempuri.org>
C: RCPT TO:<receiver+37@tempuri.org>
C: RCPT TO:<receiver+38@tempuri.org>
C: RCPT TO:<receiver+39@tempuri.org>
C: RCPT TO:<receiver+40@tempuri.org>
C: RCPT TO:<receiver+41@tempuri.org>
C: RCPT TO:<receiver+42@tempuri.org>
C: RCPT TO:<receiver+43@tempuri.org>
C: RCPT TO:<receiver+44@tempuri.org>
C: RCPT TO:<receiver+45@tempuri.org>
C: RCPT TO:<receiver+46@tempuri.org>
C: RCPT TO:<receiver+47@tempuri.org>
C: RCPT TO:<receiver+48@tempuri.org>
C: RCPT TO:<receiver+49@tempuri.org>
C: RCPT TO:<receiver+50@tempuri.org>
C: RCPT TO:<receiver+51@tempuri.org>
C: RCPT TO:<receiver+52@tempuri.org>
C: RCPT TO:<receiver+53@tempuri.org>
C: RCPT TO:<receiver+54@tempuri.org>
C: RCPT TO:<receiver+55@tempuri.org>
C: RCPT TO:<receiver+56@tempuri.org>
C: RCPT TO:<receiver+57@tempuri.org>
C: RCPT TO:<receiver+58@tempuri.org>
C: RCPT TO:<receiver+59@tempuri.org>
C: RCPT TO:<receiver+60@tempuri.org>
C: RCPT TO:<receiver+61@tempuri.org>
C: RCPT TO:<receiver+62@tempuri.org>
C: RCPT TO:<receiver+63@tempuri.org>
C: RCPT TO:<receiver+64@tempuri.org>
C: RCPT TO:<receiver+65@tempuri.org>
C: RCPT TO:<receiver+66@tempuri.org>
C: RCPT TO:<receiver+67@tempuri.org>
C: RCPT TO:<receiver+68@tempuri.org>
C: RCPT TO:<receiver+69@tempuri.org>
C: RCPT TO:<receiver+70@tempuri.org>
C: RCPT TO:<receiver+71@tempuri.org>
C: RCPT TO:<receiver+72@tempuri.org>
C: RCPT TO:<receiver+73@tempuri.org>
C: RCPT TO:<receiver+74@tempuri.org>
C: RCPT TO:<receiver+75@tempuri.org>
C: RCPT TO:<receiver+76@tempuri.org>
C: RCPT TO:<receiver+77@tempuri.org>
C: RCPT TO:<receiver+78@tempuri.org>
C: RCPT TO:<receiver+79@tempuri.org>
C: RCPT TO:<receiver+80@tempuri.org>
C: RCPT TO:<receiver+81@tempuri.org>
C: RCPT TO:<receiver+82@tempuri.org>
C: RCPT TO:<receiver+83@tempuri.org>
C: RCPT TO:<receiver+84@tempuri.org>
C: RCPT TO:<receiver+85@tempuri.org>
C: RCPT TO:<receiver+86@tempuri.org>
C: RCPT TO:<receiver+87@tempuri.org>
C: RCPT TO:<receiver+88@tempuri.org>
C: RCPT TO:<receiver+89@tempuri.org>
C: RCPT TO:<receiver+90@tempuri.org>
C: RCPT TO:<receiver+91@tempuri.org>
C: RCPT TO:<receiver+92@tempuri.org>
C: RCPT TO:<receiver+93@tempuri.org>
C: RCPT TO:<receiver+94@tempuri.org>
C: RCPT TO:<receiver+95@tempuri.org>
C: RCPT TO:<receiver+96@tempuri.org>
C: RCPT TO:<receiver+97@tempuri.org>
C: RCPT TO:<receiver+98@tempuri.org>
C: RCPT TO:<receiver+99@tempuri.org>
C: RCPT TO:<receiver+100@tempuri.org>
C: RCPT TO:<receiver+101@tempuri.org>
C: RCPT TO:<receiver+102@tempuri.org>
C: RCPT TO:<receiver+103@tempuri.org>
C: RCPT TO:<receiver+104@tempuri.org>
C: RCPT TO:<receiver+105@tempuri.org>
C: RCPT TO:<receiver+106@tempuri.org>
C: RCPT TO:<receiver+107@tempuri.org>
C: RCPT TO:<receiver+108@tempuri.org>
C: RCPT TO:<receiver+109@tempuri.org>
C: RCPT TO:<receiver+110@tempuri.org>
C: RCPT TO:<receiver+111@tempuri.org>
C: RCPT TO:<receiver+112@tempuri.org>
C: RCPT TO:<receiver+113@tempuri.org>
C: RCPT TO:<receiver+114@tempuri.org>
C: RCPT TO:<receiver+115@tempuri.org>
C: RCPT TO:<receiver+116@tempuri.org>
C: RCPT TO:<receiver+117@tempuri.org>
C: RCPT TO:<receiver+118@tempuri.org>
C: RCPT TO:<receiver+119@tempuri.org>
C: RCPT TO:<receiver+120@tempuri.org>
C: RCPT TO:<receiver+121@tempuri.org>
C: RCPT TO:<receiver+122@tempuri.org>
C: RCPT TO:<receiver+123@tempuri.org>
C: RCPT TO:<receiver+124@tempuri.org>
C: RCPT TO:<receiver+125@tempuri.org>
C: RCPT TO:<receiver+126@tempuri.org>
C: RCPT TO:<receiver+127@tempuri.org>
C: RCPT TO:<receiver+128@tempuri.org>
C: RCPT TO:<receiver+129@tempuri.org>
C: RCPT TO:<receiver+130@tempuri.org>
C: RCPT TO:<receiver+131@tempuri.org>
C: RCPT TO:<receiver+132@tempuri.org>
C: RCPT TO:<receiver+133@tempuri.org>
C: RCPT TO:<receiver+134@tempuri.org>
C: RCPT TO:<receiver+135@tempuri.org>
C: RCPT TO:<receiver+136@tempuri.org>
C: RCPT TO:<receiver+137@tempuri.org>
C: RCPT TO:<receiver+138@tempuri.org>
C: RCPT TO:<receiver+139@tempuri.org>
C: RCPT TO:<receiver+140@tempuri.org>
C: RCPT TO:<receiver+141@tempuri.org>
C: RCPT TO:<receiver+142@tempuri.org>
C: RCPT TO:<receiver+143@tempuri.org>
C: RCPT TO:<receiver+144@tempuri.org>
C: RCPT TO:<receiver+145@tempuri.org>
C: RCPT TO:<receiver+146@tempuri.org>
C: RCPT TO:<receiver+147@tempuri.org>
C: RCPT TO:<receiver+148@tempuri.org>
C: RCPT TO:<receiver+149@tempuri.org>
C: RCPT TO:<receiver+150@tempuri.org>
C: RCPT TO:<receiver+151@tempuri.org>
C: RCPT TO:<receiver+152@tempuri.org>
C: RCPT TO:<receiver+153@tempuri.org>
C: RCPT TO:<receiver+154@tempuri.org>
C: RCPT TO:<receiver+155@tempuri.org>
C: RCPT TO:<receiver+156@tempuri.org>
C: RCPT TO:<receiver+157@tempuri.org>
C: RCPT TO:<receiver+158@tempuri.org>
C: RCPT TO:<receiver+159@tempuri.org>
C: RCPT TO:<receiver+160@tempuri.org>
C: RCPT TO:<receiver+161@tempuri.org>
C: RCPT TO:<receiver+162@tempuri.org>
C: RCPT TO:<receiver+163@tempuri.org>
C: RCPT TO:<receiver+164@tempuri.org>
C: RCPT TO:<receiver+165@tempuri.org>
C: RCPT TO:<receiver+166@tempuri.org>
C: RCPT TO:<receiver+167@tempuri.org>
C: RCPT TO:<receiver+168@tempuri.org>
C: RCPT TO:<receiver+169@tempuri.org>
C: RCPT TO:<receiver+170@tempuri.org>
C: RCPT TO:<receiver+171@tempuri.org>
C: RCPT TO:<receiver+172@tempuri.org>
C: RCPT TO:<receiver+173@tempuri.org>
C: RCPT TO:<receiver+174@tempuri.org>
C: RCPT TO:<receiver+175@tempuri.org>
C: RCPT TO:<receiver+176@tempuri.org>
C: RCPT TO:<receiver+177@tempuri.org>
C: RCPT TO:<receiver+178@tempuri.org>
C: RCPT TO:<receiver+179@tempuri.org>
C: RCPT TO:<receiver+180@tempuri.org>
C: RCPT TO:<receiver+181@tempuri.org>
C: RCPT TO:<receiver+182@tempuri.org>
C: RCPT TO:<receiver+183@tempuri.org>
C: RCPT TO:<receiver+184@tempuri.org>
C: RCPT TO:<receiver+185@tempuri.org>
C: RCPT TO:<receiver+186@tempuri.org>
C: RCPT TO:<receiver+187@tempuri.org>
C: RCPT TO:<receiver+188@tempuri.org>
C: RCPT TO:<receiver+189@tempuri.org>
C: RCPT TO:<receiver+190@tempuri.org>
C: RCPT TO:<receiver+191@tempuri.org>
C: RCPT TO:<receiver+192@tempuri.org>
C: RCPT TO:<receiver+193@tempuri.org>
C: RCPT TO:<receiver+194@tempuri.org>
C: RCPT TO:<receiver+195@tempuri.org>
C: RCPT TO:<receiver+196@tempuri.org>
C: RCPT TO:<receiver+197@tempuri.org>
C: RCPT TO:<receiver+198@tempuri.org>
C: RCPT TO:<receiver+199@tempuri.org>
C: RCPT TO:<receiver+200@tempuri.org>
C: RCPT TO:<receiver+201@tempuri.org>
C: RCPT TO:<receiver+202@tempuri.org>
C: RCPT TO:<receiver+203@tempuri.org>
C: RCPT TO:<receiver+204@tempuri.org>
C: RCPT TO:<receiver+205@tempuri.org>
C: RCPT TO:<receiver+206@tempuri.org>
C: RCPT TO:<receiver+207@tempuri.org>
C: RCPT TO:<receiver+208@tempuri.org>
C: RCPT TO:<receiver+209@tempuri.org>
C: RCPT TO:<receiver+210@tempuri.org>
C: RCPT TO:<receiver+211@tempuri.org>
C: RCPT TO:<receiver+212@tempuri.org>
C: RCPT TO:<receiver+213@tempuri.org>
C: RCPT TO:<receiver+214@tempuri.org>
C: RCPT TO:<receiver+215@tempuri.org>
C: RCPT TO:<receiver+216@tempuri.org>
C: RCPT TO:<receiver+217@tempuri.org>
C: RCPT TO:<receiver+218@tempuri.org>
C: RCPT TO:<receiver+219@tempuri.org>
C: RCPT TO:<receiver+220@tempuri.org>
C: RCPT TO:<receiver+221@tempuri.org>
C: RCPT TO:<receiver+222@tempuri.org>
C: RCPT TO:<receiver+223@tempuri.org>
C: RCPT TO:<receiver+224@tempuri.org>
C: RCPT TO:<receiver+225@tempuri.org>
C: RCPT TO:<receiver+226@tempuri.org>
C: RCPT TO:<receiver+227@tempuri.org>
C: RCPT TO:<receiver+228@tempuri.org>
C: RCPT TO:<receiver+229@tempuri.org>
C: RCPT TO:<receiver+230@tempuri.org>
C: RCPT TO:<receiver+231@tempuri.org>
C: RCPT TO:<receiver+232@tempuri.org>
C: RCPT TO:<receiver+233@tempuri.org>
C: RCPT TO:<receiver+234@tempuri.org>
C: RCPT TO:<receiver+235@tempuri.org>
C: RCPT TO:<receiver+236@tempuri.org>
C: RCPT TO:<receiver+237@tempuri.org>
C: RCPT TO:<receiver+238@tempuri.org>
C: RCPT TO:<receiver+239@tempuri.org>
C: RCPT TO:<receiver+240@tempuri.org>
C: RCPT TO:<receiver+241@tempuri.org>
C: RCPT TO:<receiver+242@tempuri.org>
C: RCPT TO:<receiver+243@tempuri.org>
C: RCPT TO:<receiver+244@tempuri.org>
C: RCPT TO:<receiver+245@tempuri.org>
C: RCPT TO:<receiver+246@tempuri.org>
C: RCPT TO:<receiver+247@tempuri.org>
C: RCPT TO:<receiver+248@tempuri.org>
C: RCPT TO:<receiver+249@tempuri.org>
C: RCPT TO:<receiver+250@tempuri.org>
C: RCPT TO:<receiver+251@tempuri.org>
C: RCPT TO:<receiver+252@tempuri.org>
C: RCPT TO:<receiver+253@tempuri.org>
C: RCPT TO:<receiver+254@tempuri.org>
C: RCPT TO:<receiver+255@tempuri.org>
C: RCPT TO:<receiver+256@tempuri.org>
C: RCPT TO:<receiver+257@tempuri.org>
C: RCPT TO:<receiver+258@tempuri.org>
C: RCPT TO:<receiver+259@tempuri.org>
C: RCPT TO:<receiver+260@tempuri.org>
C: RCPT TO:<receiver+261@tempuri.org>
C: RCPT TO:<receiver+262@tempuri.org>
C: RCPT TO:<receiver+263@tempuri.org>
C: RCPT TO:<receiver+264@tempuri.org>
C: RCPT TO:<receiver+265@tempuri.org>
C: RCPT TO:<receiver+266@tempuri.org>
C: RCPT TO:<receiver+267@tempuri.org>
C: RCPT TO:<receiver+268@tempuri.org>
C: RCPT TO:<receiver+269@tempuri.org>
C: RCPT TO:<receiver+270@tempuri.org>
C: RCPT TO:<receiver+271@tempuri.org>
C: RCPT TO:<receiver+272@tempuri.org>
C: RCPT TO:<receiver+273@tempuri.org>
C: RCPT TO:<receiver+274@tempuri.org>
C: RCPT TO:<receiver+275@tempuri.org>
C: RCPT TO:<receiver+276@tempuri.org>
C: RCPT TO:<receiver+277@tempuri.org>
C: RCPT TO:<receiver+278@tempuri.org>
C: RCPT TO:<receiver+279@tempuri.org>
C: RCPT TO:<receiver+280@tempuri.org>
C: RCPT TO:<receiver+281@tempuri.org>
C: RCPT TO:<receiver+282@tempuri.org>
C: RCPT TO:<receiver+283@tempuri.org>
C: RCPT TO:<receiver+284@tempuri.org>
C: RCPT TO:<receiver+285@tempuri.org>
C: RCPT TO:<receiver+286@tempuri.org>
C: RCPT TO:<receiver+287@tempuri.org>
C: RCPT TO:<receiver+288@tempuri.org>
C: RCPT TO:<receiver+289@tempuri.org>
C: RCPT TO:<receiver+290@tempuri.org>
C: RCPT TO:<receiver+291@tempuri.org>
C: RCPT TO:<receiver+292@tempuri.org>
C: RCPT TO:<receiver+293@tempuri.org>
C: RCPT TO:<receiver+294@tempuri.org>
C: RCPT TO:<receiver+295@tempuri.org>
C: RCPT TO:<receiver+296@tempuri.org>
C: RCPT TO:<receiver+297@tempuri.org>
C: RCPT TO:<receiver+298@tempuri.org>
C: RCPT TO:<receiver+299@tempuri.org>
C: RCPT TO:<receiver+300@tempuri.org>
C: RCPT TO:<receiver+301@tempuri.org>
C: RCPT TO:<receiver+302@tempuri.org>
C: RCPT TO:<receiver+303@tempuri.org>
C: RCPT TO:<receiver+304@tempuri.org>
C: RCPT TO:<receiver+305@tempuri.org>
C: RCPT TO:<receiver+306@tempuri.org>
C: RCPT TO:<receiver+307@tempuri.org>
C: RCPT TO:<receiver+308@tempuri.org>
C: RCPT TO:<receiver+309@tempuri.org>
C: RCPT TO:<receiver+310@tempuri.org>
C: RCPT TO:<receiver+311@tempuri.org>
C: RCPT TO:<receiver+312@tempuri.org>
C: RCPT TO:<receiver+313@tempuri.org>
C: RCPT TO:<receiver+314@tempuri.org>
C: RCPT TO:<receiver+315@tempuri.org>
C: RCPT TO:<receiver+316@tempuri.org>
C: RCPT TO:<receiver+317@tempuri.org>
C: RCPT TO:<receiver+318@tempuri.org>
C: RCPT TO:<receiver+319@tempuri.org>
C: RCPT TO:<receiver+320@tempuri.org>
C: RCPT TO:<receiver+321@tempuri.org>
C: RCPT TO:<receiver+322@tempuri.org>
C: RCPT TO:<receiver+323@tempuri.org>
C: RCPT TO:<receiver+324@tempuri.org>
C: RCPT TO:<receiver+325@tempuri.org>
C: RCPT TO:<receiver+326@tempuri.org>
C: RCPT TO:<receiver+327@tempuri.org>
C: RCPT TO:<receiver+328@tempuri.org>
C: RCPT TO:<receiver+329@tempuri.org>
C: RCPT TO:<receiver+330@tempuri.org>
C: RCPT TO:<receiver+331@tempuri.org>
C: RCPT TO:<receiver+332@tempuri.org>
C: RCPT TO:<receiver+333@tempuri.org>
C: RCPT TO:<receiver+334@tempuri.org>
C: RCPT TO:<receiver+335@tempuri.org>
C: RCPT TO:<receiver+336@tempuri.org>
C: RCPT TO:<receiver+337@tempuri.org>
C: RCPT TO:<receiver+338@tempuri.org>
C: RCPT TO:<receiver+339@tempuri.org>
C: RCPT TO:<receiver+340@tempuri.org>
C: RCPT TO:<receiver+341@tempuri.org>
C: RCPT TO:<receiver+342@tempuri.org>
C: RCPT TO:<receiver+343@tempuri.org>
C: RCPT TO:<receiver+344@tempuri.org>
C: RCPT TO:<receiver+345@tempuri.org>
C: RCPT TO:<receiver+346@tempuri.org>
C: RCPT TO:<receiver+347@tempuri.org>
C: RCPT TO:<receiver+348@tempuri.org>
C: RCPT TO:<receiver+349@tempuri.org>
C: RCPT TO:<receiver+350@tempuri.org>
C: RCPT TO:<receiver+351@tempuri.org>
C: RCPT TO:<receiver+352@tempuri.org>
C: RCPT TO:<receiver+353@tempuri.org>
C: RCPT TO:<receiver+354@tempuri.org>
C: RCPT TO:<receiver+355@tempuri.org>
C: RCPT TO:<receiver+356@tempuri.org>
C: RCPT TO:<receiver+357@tempuri.org>
C: RCPT TO:<receiver+358@tempuri.org>
C: RCPT TO:<receiver+359@tempuri.org>
C: RCPT TO:<receiver+360@tempuri.org>
C: RCPT TO:<receiver+361@tempuri.org>
C: RCPT TO:<receiver+362@tempuri.org>
C: RCPT TO:<receiver+363@tempuri.org>
C: RCPT TO:<receiver+364@tempuri.org>
C: RCPT TO:<receiver+365@tempuri.org>
C: RCPT TO:<receiver+366@tempuri.org>
C: RCPT TO:<receiver+367@tempuri.org>
C: RCPT TO:<receiver+368@tempuri.org>
C: RCPT TO:<receiver+369@tempuri.org>
C: RCPT TO:<receiver+370@tempuri.org>
C: RCPT TO:<receiver+371@tempuri.org>
C: RCPT TO:<receiver+372@tempuri.org>
C: RCPT TO:<receiver+373@tempuri.org>
C: RCPT TO:<receiver+374@tempuri.org>
C: RCPT TO:<receiver+375@tempuri.org>
C: RCPT TO:<receiver+376@tempuri.org>
C: RCPT TO:<receiver+377@tempuri.org>
C: RCPT TO:<receiver+378@tempuri.org>
C: RCPT TO:<receiver+379@tempuri.org>
C: RCPT TO:<receiver+380@tempuri.org>
C: RCPT TO:<receiver+381@tempuri.org>
C: RCPT TO:<receiver+382@tempuri.org>
C: RCPT TO:<receiver+383@tempuri.org>
C: RCPT TO:<receiver+384@tempuri.org>
C: RCPT TO:<receiver+385@tempuri.org>
C: RCPT TO:<receiver+386@tempuri.org>
C: RCPT TO:<receiver+387@tempuri.org>
C: RCPT TO:<receiver+388@tempuri.org>
C: RCPT TO:<receiver+389@tempuri.org>
C: RCPT TO:<receiver+390@tempuri.org>
C: RCPT TO:<receiver+391@tempuri.org>
C: RCPT TO:<receiver+392@tempuri.org>
C: RCPT TO:<receiver+393@tempuri.org>
C: RCPT TO:<receiver+394@tempuri.org>
C: RCPT TO:<receiver+395@tempuri.org>
C: RCPT TO:<receiver+396@tempuri.org>
C: RCPT TO:<receiver+397@tempuri.org>
C: RCPT TO:<receiver+398@tempuri.org>
C: RCPT TO:<receiver+399@tempuri.org>
C: RCPT TO:<receiver+400@tempuri.org>
C: RCPT TO:<receiver+401@tempuri.org>
C: RCPT TO:<receiver+402@tempuri.org>
C: RCPT TO:<receiver+403@tempuri.org>
C: RCPT TO:<receiver+404@tempuri.org>
C: RCPT TO:<receiver+405@tempuri.org>
C: RCPT TO:<receiver+406@tempuri.org>
C: RCPT TO:<receiver+407@tempuri.org>
C: RCPT TO:<receiver+408@tempuri.org>
C: RCPT TO:<receiver+409@tempuri.org>
C: RCPT TO:<receiver+410@tempuri.org>
C: RCPT TO:<receiver+411@tempuri.org>
C: RCPT TO:<receiver+412@tempuri.org>
C: RCPT TO:<receiver+413@tempuri.org>
C: RCPT TO:<receiver+414@tempuri.org>
C: RCPT TO:<receiver+415@tempuri.org>
C: RCPT TO:<receiver+416@tempuri.org>
C: RCPT TO:<receiver+417@tempuri.org>
C: RCPT TO:<receiver+418@tempuri.org>
C: RCPT TO:<receiver+419@tempuri.org>
C: RCPT TO:<receiver+420@tempuri.org>
C: RCPT TO:<receiver+421@tempuri.org>
C: RCPT TO:<receiver+422@tempuri.org>
C: RCPT TO:<receiver+423@tempuri.org>
C: RCPT TO:<receiver+424@tempuri.org>
C: RCPT TO:<receiver+425@tempuri.org>
C: RCPT TO:<receiver+426@tempuri.org>
C: RCPT TO:<receiver+427@tempuri.org>
C: RCPT TO:<receiver+428@tempuri.org>
C: RCPT TO:<receiver+429@tempuri.org>
C: RCPT TO:<receiver+430@tempuri.org>
C: RCPT TO:<receiver+431@tempuri.org>
C: RCPT TO:<receiver+432@tempuri.org>
C: RCPT TO:<receiver+433@tempuri.org>
C: RCPT TO:<receiver+434@tempuri.org>
C: RCPT TO:<receiver+435@tempuri.org>
C: RCPT TO:<receiver+436@tempuri.org>
C: RCPT TO:<receiver+437@tempuri.org>
C: RCPT TO:<receiver+438@tempuri.org>
C: RCPT TO:<receiver+439@tempuri.org>
C: RCPT TO:<receiver+440@tempuri.org>
C: RCPT TO:<receiver+441@tempuri.org>
C: RCPT TO:<receiver+442@tempuri.org>
C: RCPT TO:<receiver+443@tempuri.org>
C: RCPT TO:<receiver+444@tempuri.org>
C: RCPT TO:<receiver+445@tempuri.org>
C: RCPT TO:<receiver+446@tempuri.org>
C: RCPT TO:<receiver+447@tempuri.org>
C: RCPT TO:<receiver+448@tempuri.org>
C: RCPT TO:<receiver+449@tempuri.org>
C: RCPT TO:<receiver+450@tempuri.org>
C: RCPT TO:<receiver+451@tempuri.org>
C: RCPT TO:<receiver+452@tempuri.org>
C: RCPT TO:<receiver+453@tempuri.org>
C: RCPT TO:<receiver+454@tempuri.org>
C: RCPT TO:<receiver+455@tempuri.org>
C: RCPT TO:<receiver+456@tempuri.org>
C: RCPT TO:<receiver+457@tempuri.org>
C: RCPT TO:

**Comment by mhjelmseth** [comment]
Great work @jstedfast 👍 
Do you know when you're going to release a NuGet package with this fix included?

**Comment by jstedfast** [comment]
😱 

Thanks for reminding me! I'll try to make a new release this weekend.

**Comment by jstedfast** [comment]
I released MailKit v3.5.0 with this fix.
